velodrome/transform/transform.go

```diff
@@ -88,28 +128,24 @@ func runProgram(config *transformConfig) error {
 		if config.once {
 			break
 		}
-
 		<-ticker
 	}
 
 	return nil
-
 }
 
 func main() {
 	config := &transformConfig{}
 	root := &cobra.Command{
 		Use:   filepath.Base(os.Args[0]),
 		Short: "Transform sql database info into influx stats",
-		RunE: func(_ *cobra.Command, _ []string) error {
-			return runProgram(config)
-		},
 	}
-	addRootFlags(root, config)
-	CheckRootFlags(config)
+	config.AddFlags(root)
 	config.MySQLConfig.AddFlags(root)
 	config.InfluxConfig.AddFlags(root)
 
+	// root.AddCommand(plugins.YourPlugin(config.run))
```

Comment:
This is how one is supposed to register his plugin. There are no plugins yet, so that's a placeholder example.